charts/attribute-masking-service/templates/_helpers.tpl

```diff
@@ -102,7 +102,7 @@ Calculate a storage path based on the code release artifact id or the supplied v
 Calculate a storage name based on the code release artifact id or the supplied value of codeRelease
```

Comment:
Found two lingering references in the code to codeRelease: /templates/_helpers.tpl;  create-kafka-queues/src/main/resources/values.yaml